tests/foreman/ui/test_contentview.py

```diff
@@ -3766,31 +3720,23 @@ def test_positive_remove_cv_version_from_multi_env_capsule_scenario(self):
         @Steps:
 
         1. Create a content view
-
         2. Setup satellite to use a capsule and to sync all lifecycle
```

Comment:
Usually, its not a common practice to sync all environments to capsule and especially the `Library` as it includes all contents. I think we need to revisit the test once we will have implementation around capsule in place.